src/js/bootstrap.js

```diff
@@ -0,0 +1,4153 @@
+/*!
+ * Bootstrap v4.5.0 (https://getbootstrap.com/)
+ * Copyright 2011-2020 The Bootstrap Authors (https://github.com/twbs/bootstrap/graphs/contributors)
+ * Licensed under MIT (https://github.com/twbs/bootstrap/blob/master/LICENSE)
+ */
+!(function (t, e) {
+  "object" == typeof exports && "undefined" != typeof module
+    ? e(exports, require("jquery"))
+    : "function" == typeof define && define.amd
+    ? define(["exports", "jquery"], e)
+    : e(((t = t || self).bootstrap = {}), t.jQuery);
```

Comment:
Codacy found an issue: [Avoid assignments in operands](https://app.codacy.com/manual/hello_100/jca-ui-kit/pullRequest?prid=5731813)